src/main/java/com/paloaltonetworks/osc/api/PANManagerSecurityGroupApi.java

```diff
@@ -51,12 +51,12 @@ public String createSecurityGroup(String name, String iscId, SecurityGroupMember
          * Check if tag exists - if not create
          */
         boolean tagExists;
-        String status;
+        String status = null;
         String pan_serial = null;
 
-        tagExists = this.showOperations.TagExists(name);
+        tagExists = this.client.policyTagExists(name);
```

Comment:
should this be this.client.tagExists ? we dont care if there is a policy tag with the same name.

Reply:
This pull request was about PolicyApi. I am working on the other Apis now.

Reply:
its hard to figure out from the pull request which pieces of the change map to policy api.

looks like we have made changes which impact other API's as well. 

We want to merge pull request which are functionally complete. If you are in the middle of other changes for commiting config,device groups etc we should either back them out or fix the issues.